mobros/types/apt_cache_singleton.py

```diff
@@ -25,6 +25,10 @@ def __new__(cls):
                 logging.warning(
                     "Unable to do apt update. Please run as sudo, or execute it before mobros!"
                 )
+            except apt.cache.FetchFailedException:
+                logging.error(
+                    "Unable to fetch apt cache. Please check your internet connection! Try running 'sudo apt update' for more info."
```

Comment:
I believe the local apt cache can also be interrogated without need for an Internet connection 
- local cache can come from previous apt operations (update, download, install, ...)
- ubuntu distribution usb / cdrom / folder